index.d.ts

```diff
@@ -295,6 +300,17 @@ declare namespace execa {
 		Promise<ExecaReturnValue<StdoutErrorType>>;
 }
 
+interface killOptions {
```

Comment:
The interface should be defined before it's used and should be `KillOptions`, and should be inside the `execa` namespace.

Reply:
Fixed: 465a4d1d9fc7b640c142a00aa6ed08dc0c8a7b15